node_io/test/file_stat_test.dart

```diff
@@ -18,5 +19,19 @@ void main() {
       expect(stat.size, isNotNull);
       expect(stat.mode, isNotNull);
     });
+
+    test('stat_directory', () async {
+      var stat = await FileStat.stat(Directory.current.path);
+      expect(stat.accessed, new isInstanceOf<DateTime>());
+      expect(stat.changed, new isInstanceOf<DateTime>());
+      expect(stat.modified, new isInstanceOf<DateTime>());
```

Comment:
Just a quick note: I just published an update of all node interop packages with upgrade to `test: ^1.0.0` which deprecated `isInstanceOf` for some unfortunate reason.

These would need to be replaced with `const TypeMatcher<Type>()`.

Reply:
ok